lvsfunc/comparison.py

```diff
@@ -536,6 +541,23 @@ def tvbd_diff(tv: vs.VideoNode, bd: vs.VideoNode,
     if not frames:
         raise ValueError("tvbd_diff: no differences found")
 
+    if write_log:
+        from itertools import groupby, count
```

Comment:
itertools is pretty standard so there's not really a reason to defer this import rather than putting it at the top of the py file